index.rst

```diff
@@ -138,9 +138,10 @@ Hardpoint tests
 
 * `SITCOMTN-082 <https://sitcomtn-082.lsst.io/>`_ **M1M3 Hardpoint Breakaway** 
 
-  *Results*:  Breakaway system works in general. Some notable exceptions (HP2, HP5 breakaway system faults at low (<30 deg) elevations, response shape also different) seem to be have an explanation according to Yijung and Petr.
+  *Results*:  Breakaway system works in general. Some notable exceptions (HP2, HP5 breakaway system faults at low (<30 deg) elevations, response shape also different) seem to be have an explanation according to Yijung and Petr. The format of the technical notes is in the new format (SITCOM-1111-2 dated 2024-01-23). However, these updates are not shown in the “Current” version which is in the old format. The technical note SITCOM-111-2 contains more explanations and conclusions than the current one.
```

Comment:
Can you check with the authors why is this the case? (why technote 82 does not appear updated to the new format?)

Reply:
Yijung kang is the author who appears in the tecnhote with the new format ( https://sitcomtn-082.lsst.io/v/SITCOM-1111-2/index.html). However, I am not sure if this author is the one who changed the technote format. I can't see who is the author that migrated the tecnote from the old format to then new one because  there is no associated ticket